src/components/FilterableTable.react.js

```diff
@@ -0,0 +1,45 @@
+import {FilterBarActor} from "../actors/FilterBarActor";
+import {TableActor} from "../actors/TableActor";
+
+import {FilterBarStore} from "../stores/FilterBarStore";
+import {TableStore} from "../stores/TableStore";
+
+import {FilterBar} from "./FilterBar/FilterBar.react";
+import {Table} from "./Table/Table.react";
+
+export class FilterableTable extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.filterBarStore = new FilterBarStore(props.filterBarConfiguration);
+    this.tableStore = new TableStore(props.tableConfiguration);
+
+    this.filterBarActor = new FilterBarActor(this.filterBarStore, this.tableStore);
+    this.tableActor = new TableActor(this.filterBarStore, this.tableStore);
+  }
+
+  getChildContext() {
```

Comment:
I wasn't aware that React had made `getChildContext` and all the context stuff generally part of their public API. Is it documented somewhere, or are we relying on an undocumented feature?